winlogbeat/docs/modules.asciidoc

```diff
@@ -22,6 +22,9 @@ a `script` processor to your configuration file and point it at the included
 script file for the module. The documentation for each module includes an
 example.
 
+NOTE: The provided modules only support events in english. For more information
```

Comment:
```suggestion
NOTE: The provided modules only support events in English. For more information
```

Reply:
Do we know what will break if a non-English language is used?

Reply:
I am doing some more testing around this, but from https://github.com/elastic/beats/issues/14209 and some support tickets, it seems separators and delimiters are different depending on the language, breaking the dissects in some cases.